flux/stdlib/influxdata/influxdb/rules.go

```diff
@@ -255,12 +243,6 @@ func (rule PushDownReadTagKeysRule) Rewrite(ctx context.Context, pn plan.Node) (
 	// constructing our own replacement. We do not care about it
 	// at the moment though which is why it is not in the pattern.
 
-	// The tag keys mechanism doesn't know about fields so we cannot
-	// push down _field comparisons in 1.x.
-	if fromSpec.Predicate != nil && hasFieldRef(fromSpec.Predicate.Root) {
```

Comment:
What changed here?

Reply:
AFAICS the original bug was fixed in Flux/OSS. I found a reproducer in #19561, I'll add it as a test case to be sure.

Reply:
I'm going to back out the changes to the `ReadTags` pushdowns for now. I think there might be more differences lurking between 1.x and 2.x, and want to keep this PR focused on the aggregate pushdowns.